utils/multiplatform/src/commonMain/kotlin/com/bumble/appyx/utils/multiplatform/Logger.kt

```diff
@@ -0,0 +1,5 @@
+package com.bumble.appyx.utils.multiplatform
+
+expect class Logger() {
```

Comment:
FYI we have a class `AppyxLogger`

Reply:
ah good :)

Reply:
refactored it to move it to multiplatform common